camera_matrix:
  cols: 3
  data:
  - 2117.563554264914
  - 0.0
  - 511.1683061676541
  - 0.0
  - 2827.460416482567
  - 239.51363095347534
  - 0.0
  - 0.0
  - 1.0
  rows: 3
camera_name: sony_1
distortion_coefficients:
  cols: 5
  data:
  - 0.20892062544517467
  - -8.132966434625413
  - -0.01901850474216138
  - 0.008197489003613376
  - 0.0
  rows: 1
distortion_model: plumb_bob
image_height: 480
image_width: 640
projection_matrix:
  cols: 4
  data:
  - 2077.690185546875
  - 0.0
  - 513.4661217259236
  - 0.0
  - 0.0
  - 2827.843994140625
  - 237.66461150324903
  - 0.0
  - 0.0
  - 0.0
  - 1.0
  - 0.0
  rows: 3
rectification_matrix:
  cols: 3
  data:
  - 1.0
  - 0.0
  - 0.0
  - 0.0
  - 1.0
  - 0.0
  - 0.0
  - 0.0
  - 1.0
  rows: 3
